### PR TITLE
Snip tool to split strokes

### PR DESCRIPTION
**Pull request by andybak**
Seems to work ok enough to push out to our brave beta testers.

### CODE REVIEW

**Comment by andybak** [comment]
This is just the snip tool for now - more to come hopefully.

File: Assets/Scripts/Commands/SnipStrokeCommand.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
+﻿// Copyright 2020 The Tilt Brush Authors
```
Comment (mikeage) [inline Assets/Scripts/Commands/SnipStrokeCommand.cs:1]: 2022, Open Brush

**Review by mikeage — COMMENTED** [review]
Cool! 

Maybe rename the PR to mention that this is the snip tool; we'll have more coming soon

File: Assets/Scripts/Commands/SnipStrokeCommand.cs (hunk 1 above)
Comment (andybak) [reply Assets/Scripts/Commands/SnipStrokeCommand.cs:1]: We so need to automate this!

Comment (mikeskydev) [reply Assets/Scripts/Commands/SnipStrokeCommand.cs]: I was looking through the codebase and I think we missed a few elsewhere too 😅

Comment (mikeage) [reply Assets/Scripts/Commands/SnipStrokeCommand.cs]: I thought we were only going to do new code. I don't mind doing a global search and replace if desired

Comment (mikeskydev) [reply Assets/Scripts/Commands/SnipStrokeCommand.cs]: Ah, I mean on new file commits from the Open Brush authors. Gotta leave the credit to TB team!

**Pull request merged**